Dockerfile

```diff
@@ -0,0 +1,18 @@
+FROM python:3.8-alpine
+
+ENV LANG=C.UTF-8
+
+RUN apk update && apk add --no-cache alpine-sdk \ 
+                                     python3-dev \
+                                     libffi-dev \
+                                     rust \  
+                                     cargo \
```

Comment:
Can I know why do we need `rust (+ cargo)` here?

Reply:
I am sorry, it is my mistake.
I am fixed this issue.